modules/main/src/main/env/service/entrypoint.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
```

Comment:
for completeness, https://github.com/intuit/wasabi/pull/31 is where the dockerfile is that uses this script